### PR TITLE
Shadowling veil nullifies glowy for SOME TIME and enthrall removes it for ALL TIME

### PR DESCRIPTION
**Pull request by Sniblet**
# Document the changes in your pull request

Veil turns off glowy's light for 40 seconds. If someone is enthralled, their glowy is removed

# Changelog

:cl:  
tweak: Veil turns off glowy's light for 40 seconds. If someone is enthralled, their glowy is removed
/:cl:

### CODE REVIEW

**Comment by RG4ORDR** [comment]
Finally the glowy genetics monopoly is being torn down.

**Comment by GraveHat** [comment]
You should close that other PR since this is objectively better.

**Comment by MajManatee** [comment]
120 is an AWFULLY long time
surely 60 is more reasonable

**Comment by Sniblet** [comment]
60 is shorter than thralling a mindshielded person
70 means you have to get that mindshielded person to a safe area very fast
80 is alright...
But I mean, if you're smart at all you have other lights. What does it matter?

**Review by Moltijoe — APPROVED** [review]
2 minutes seems good, it might end up being too long, but it can always be changed in a later pr if that is found to be the case

**Comment by Therandomhoboo** [comment]
2 minutes is too long, considering veil takes 20 seconds to refresh... you might as well just remove glowy at this point.

Make it literately 5 seconds shorter than the veil cd or just make a PR to remove glowy.

However, I agree with the glowy being removed from thrall victims because its dumb that a thralled person can still kill you if they have glowy

**Comment by Moltijoe** [comment]
> 60 is shorter than thralling a mindshielded person 70 means you have to get that mindshielded person to a safe area very fast 80 is alright... But I mean, if you're smart at all you have other lights. What does it matter?

could always tag team a mindshielded person, one thralls, the other veils

**Comment by Therandomhoboo** [comment]
> > 60 is shorter than thralling a mindshielded person 70 means you have to get that mindshielded person to a safe area very fast 80 is alright... But I mean, if you're smart at all you have other lights. What does it matter?
> 
> could always tag team a mindshielded person, one thralls, the other veils

The main problem I find with this PR is that it ignores the fact that you have two other teammates that you work with. Yeah they can suck, but its a teamwork round. You can also convert whilst also glaring/veiling without breaking conversion I think (Or it works with glare at least)
If this gets bumped up to 120 seconds it just tilts it too heavily in the lings favour and makes glowy completely useless.

**Comment by Sniblet** [comment]
Fuck you for phrasing your point in a way I could understand
Slings may have to use veil twice for non-mindshielded glowies, and will have to for mindshielded glowies
This makes glowy Useful:tm: because casting veil more means it's on cooldown more which means slings are vulnerable more
I'm not going to make them tag-team every CIA operative they want to convert, that's a Psychotic Take:tm: and does not improve the gamemode

**Comment by CMOisLing** [comment]
Make the thralling process refresh the timer

**Comment by Therandomhoboo** [comment]
I would argue 40 seconds is still high, but this is significantly better and I would be fine at least seeing how it works out if merged.

**Comment by Sniblet** [comment]
Bearing in mind that Veil's cooldown is 12 seconds, I could theoretically go lower without infringing my moral responsibility to not make shadowling miserable to play, but I will not go lower

**Comment by Therandomhoboo** [comment]
> Bearing in mind that Veil's cooldown is 12 seconds, I could theoretically go lower without infringing my moral responsibility to not make shadowling miserable to play, but I will not go lower

Sorry to keep posting/replying. I would say 25 seconds minimum maybe but 40 is far better in my eyes to see how it works out.

**Comment by Simplehorror** [comment]
this makes glowy useless in shadowling rounds, you may as well remove it. your other PR was less of a nerf than this, because you could inject it back

**Comment by Simplehorror** [comment]
also this is powercreep after the 5 seconds removal

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -337,6 +337,11 @@
 		target.visible_message(span_big("[target] looks to have experienced a revelation!"), \
 							   span_warning("False faces all d<b>ark not real not real not--</b>"))
 		target.setOxyLoss(0) //In case the shadowling was choking them out
+		if(iscarbon(target))
```
Comment (Simplehorror) [inline yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:340]: this is going to let people know who thralls are

Comment (Simplehorror) [inline yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:340]: this is going to let people know who thralls are (you accidentally resolved this)

**Review by Simplehorror — CHANGES_REQUESTED** [review]

**Review by Simplehorror — CHANGES_REQUESTED** [review]
this is going to let people know who thralls are (you accidentally resolved this) (AGAIN)

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm (hunk 1 above)
Comment (ReddicusDragon) [reply yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:340]: it IS going to reveal who the thralls are if they all had glowy, but the issue also is, without removing glowy, we fuck over the slings

Comment (Simplehorror) [reply yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:340]: if i was a sling i'd rather them keep glowy and just not interact with them honestly

**Comment by Sniblet** [comment]
Glowy can be replaced. There are other ways to identify thralls. I don’t care.

**Comment by Therandomhoboo** [comment]
> this is going to let people know who thralls are (you accidentally resolved this) (AGAIN)

Like no one checks for the eyes as soon as lings are called. Honestly, its just minor at this point

**Comment by Simplehorror** [comment]
it should be blatantly obvious who is a fucking thrall or not I think, I'll make a PR for that

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**